### ISSUE TITLE
Cannot enter friend code

### ISSUE THREAD

**Issue by Choms** [NoMansSkyAssistant/General#9]
Hi, on the app, both in iOS and android, I cannot input the friend code (literally, on the "Friend Code" box, it won't write any keypress), thus I cannot add mine. Also great app, maybe you could consider opensourcing the code and more people could contribute to it :)

**Comment by Khaoz-Topsy** [comment]
Thank you for letting me know! I think I know what caused it, I recently updated to using a newer version of Flutter and there were some changes I needed to make, thought I caught everything 😅.
I have definitely thought of making the whole app open source instead of just parts of it. But I will need to make a whole new repo because at the moment the repo has history of the Android keystore and the Google Credential file 😅. If you would like access to the source code I'd be happy to give it to you. Send an email to me at nms@kurtlourens.com and I will add that email to the Azure DevOps team 😁
